catkit2/services/accufiz_interferometer/accufiz_interferometer.py

```diff
@@ -0,0 +1,151 @@
+import os
+import h5py
+import time
+import requests
+import uuid
+import numpy as np
+from scipy import ndimage
+import math
+from astropy.io import fits
+from glob import glob
+from catkit2.testbed.service import Service
+from catkit2.testbed.service_proxy import ServiceProxy
+from catkit2.utils import rotate_and_flip_image
+
+@ServiceProxy.register_service_interface('accufiz_interferometer')
```

Comment:
Would the cleanest course of action be for me to make this a ServiceProxy class and move this to from the services to the proxies folder? Is there any advantage of having a service over a serviceproxy?

Reply:
This is the service, and it's fine as it is (I haven't looked in details yet, but looks good overall). On top of that, you'll need to make a proxy. The service runs on its own process in a real time fashion. To prevent overloads that do not require real time, we use proxies that run on the client side, that are basically the interface with the service. The proxy stores the config so it doesn't have to be communicated through tcp at every call for example, or stores some convenience functions that do more complex computation.

Reply:
Ok - I have made the accufiz_interferometer inherit the Service class and it also implements the Camera ServiceProxy. See updated test and service.yml examples.